lib/kernel/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ acos.cl
 acosh.cl
 acospi.cl
 add_sat.cl
+addrspace_operators.ll
```

Comment:
Also note the new strategy in https://github.com/pocl/pocl/commit/4500774f4af2a134e027169b199d3dcd7195cf20 for SLEEF which may or may not be broken.
I'm also not sure if this file is used in non-"host" backends where it might cause issues now.